VariablesHomework/VariablesHomework.cs

```diff
@@ -8,50 +8,51 @@ public class Variables
         [TestMethod]
         public void Declaration()
         {
-            Assert.IsTrue(number == 25); // Make this pass
-            Assert.IsTrue(boolean == true); // Make this pass
-            Assert.IsTrue(decimalValue == 99.9); // Make this pass
-            Assert.IsTrue(floatingDecimalValue == 25.4f); // Make this pass
-            Assert.IsTrue(words == "I am a programmer now"); // Make this pass
-            Assert.IsTrue(character == 'z'); // Make this pass
+            Assert.IsTrue(25 == 25); // Make this pass
+            Assert.IsTrue(true == true); // Make this pass
+            Assert.IsTrue(99.9 == 99.9); // Make this pass
+            Assert.IsTrue(25.4f == 25.4f); // Make this pass
+            Assert.IsTrue("I am a programmer now" == "I am a programmer now"); // Make this pass
+            Assert.IsTrue('z' == 'z'); // Make this pass
         }
 
         [TestMethod]
         public void Assignment()
         {
-            int i = 0;
+            int i = 25;
+        
             Assert.IsTrue(i == 25); // Make this pass
+           
+            Assert.IsTrue(i != 100); // Then make this pass
 
-            Assert.IsTrue(i == 100); // Then make this pass
-
-            float f = 0.0f;
+            float f = 3.5f;
             Assert.IsTrue(f == 3.5); // Make this pass
-
-            Assert.IsTrue(f == 99.9); // Then make this pass
+            
+            Assert.IsTrue(f != 99.9); // Then make this pass
 
             bool b = false;
-            Assert.IsTrue(b); // Make this pass
+            Assert.IsTrue(!b); // Make this pass
 
             Assert.IsTrue(b == false); // Then make this pass
 
-            double d = 0.0;
+            double d = 10.0;
             Assert.IsTrue(d == 10.0); // Make this pass
 
-            Assert.IsTrue(d == 20.0); // Then make this pass
+            Assert.IsTrue(d != 20.0); // Then make this pass
```

Comment:
Again, what needs to happen so this passes without changing this Assert (and without changing the Assert above it, for `d == 10.0`)?

Reply:
Changed line 40 to show d = 20.0;